### DISCUSSION TITLE
Bug: Add-On options are not saved

### DISCUSSION THREAD

**Discussion by ManuelBoss** [kewisch/gdata-provider#411] [Support] [answered]
To solve the [problem with html tags in event body](https://github.com/kewisch/gdata-provider/discussions/392), I tried to enable the option "force use of plain text in event descriptions".
![grafik](https://user-images.githubusercontent.com/59780024/157085558-1e7a2392-76e2-4ff1-94fb-d26a61a19042.png)
Unfortunately, the setting disables itself on exiting the options menu of the add-on automatically.
Am I the only one with ths problem, or shall I [file an issue](https://github.com/kewisch/gdata-provider/issues).

**Comment by kewisch** [comment, marked as answer]
I've fixed this in 98a90ee8858d63d6108821184c070052e04310b9 and will be releasing 102.0.0 covering this soon.
